vnext/ReactWindows.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29009.5
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.705
```

Comment:
>15 [](start = 22, length = 2)

we should not be lowering version

Reply:
Reverted.

Reply:
Uhh I am using Visual Studio 2017 and it makes this change every time. I am upgrading to VS 2019 and will make sure this change is not checked in.

---
In reply to: [298773689](https://github.com/microsoft/react-native-windows/pull/2677#discussion_r298773689) [](ancestors = 298773689)